fdbrpc/include/fdbrpc/DDSketch.h

```diff
@@ -246,13 +246,13 @@ class DDSketchSlow : public DDSketchBase<DDSketchSlow<T>, T> {
 		this->setBucketSize(2 * offset);
 	}
 
-	size_t getIndex(T sample) { return ceil(log(sample) / logGamma) + offset; }
+	int getIndex(T sample) { return ceil(log(sample) / logGamma) + offset; }
```

Comment:
Happy to use a signed value here but we should use a type that can contain all possible index values `[0,UINT_MAX]`.

Reply:
Did some thinking after this -- I think we should assert that `errorGaurantee` will never be small enough to overflow an `int`.